languages/python3/template/requirements.txt

```diff
@@ -0,0 +1 @@
+algorithmia>=1.0.0,<2.0
```

Comment:
i think we still need `six` (it's in the pipe script) unless that's bundled with python 3.7 now?

Reply:
I actually thought it was since it imports fine on my system and according to my terminal history I never installed it. The official standard library (https://docs.python.org/3/library/) doesn't seems to have it though so I'll add it back, thanks for the catch